NGit.Test/NGit.Api/ApplyCommandTest.cs

```diff
@@ -78,10 +90,10 @@ private ApplyResult Init(string name, bool preExists, bool postExists)
 			}
 			if (postExists)
 			{
-				b = new RawText(ReadFile(name + "_PostImage"));
+				b = new RawText(ReadFile(name + "_PostImage", a != null && a.GetLineDelimiter() == "\r\n"));
```

Comment:
Looks like this is fixed in https://github.com/red-gate/ngit/commit/829b04419ca3688c28e3f7039a4b2defc245ed9e :+1: